docs/security/seccomp.md

```diff
@@ -0,0 +1,64 @@
+<!-- [metadata]>
++++
+title = "Seccomp security profiles for Docker"
+description = "Enabling seccomp in Docker"
+keywords = ["seccomp, security, docker, documentation"]
++++
+<![end-metadata]-->
+
+Seccomp security profiles for Docker
+------------------------------------
+
+The seccomp() system call operates on the Secure Computing (seccomp)
+state of the calling process.
+
+This operation is available only if the kernel is configured
```

Comment:
yes it is i can update

On Fri, Dec 4, 2015 at 11:04 AM, moxiegirl <notifications@github.com> wrote:

> In docs/security/seccomp.md
> <https://github.com/docker/docker/pull/17989#discussion_r46718340>:
>
> > @@ -0,0 +1,64 @@
> > +<!-- [metadata]>
> > ++++
> > +title = "Seccomp security profiles for Docker"
> > +description = "Enabling seccomp in Docker"
> > +keywords = ["seccomp, security, docker, documentation"]
> > ++++
> > +<![end-metadata]-->
> > +
> > +Seccomp security profiles for Docker
> > +------------------------------------
> > +
> > +The seccomp() system call operates on the Secure Computing (seccomp)
> > +state of the calling process.
> > +
> > +This operation is available only if the kernel is configured
>
> Is this a reference to the Docker Host kernel? It isn't clear.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/17989/files#r46718340>.
>